docs/topics/loaders.rst

```diff
@@ -21,26 +21,13 @@ Item Loaders are designed to provide a flexible, efficient and easy mechanism
 for extending and overriding different field parsing rules, either by spider,
 or by source format (HTML, XML, etc) without becoming a nightmare to maintain.
 
+.. note:: Item Loaders used to be a part of ``scrapy``, but moved to a self-contained library.
+    This implementation is an extension of itemloaders_ library,
+    keeping support to :ref:`Items <topics-items>` and :ref:`Response <topics-request-response>`.
+
```

Comment:
I think this should be written more in line with the counterpart note in `selectors.rst`, talking only about the present, not the past.